Sync/TabsPayload.swift

```diff
@@ -108,7 +108,7 @@ open class TabsPayload: CleartextPayloadJSON {
     // at which point the only distinction between the two is that RemoteTab is "simple" and
     // lives in Storage, and Tab is more closely tied to TabsPayload.
 
-    var remoteTabs: [RemoteTab] {
+    var remoteTabs: [RemoteTabRecord] {
```

Comment:
I wonder if you can use a type alias for `RemoteTabRecord` to `RemoveTab`? It would make the patch smaller and would mean things like comments remain accurate (eg, the couple of comments above this now refer to a non-existing type, which isn't a big deal as the reader probably knows what is meant, but still maybe worth considering?)

Reply:
That's fair. I'm not sure if this is what you meant but . . . since this stems from the swift `RemoteTab` conflicting with the one we expose in the UDL, I renamed the UDL struct and aliased it back to `RemoteTab` for android in AS (similar to what I did for `DeviceType` in order to avoid an AC breaking change). I'm open to feedback here because while these AS changes are a little hacky and hopefully short-term, they'll probably be more permanent than I'd like.